onnx/defs/function.cc

```diff
@@ -94,4 +104,32 @@ FunctionBuilderRegistry& FunctionBuilderRegistry::OnnxInstance() {
   return func_builder_registry;
 }
 
+Common::Status FunctionNewNodeHelper(
```

Comment:
This is a common nodeproto construction. You may change it with a general name, say, BuildNode or... Make sense?